### PR TITLE
Docs completion

### PR DESCRIPTION
**Pull request by kennykarnama**
Docs completion for several classes :smile: 
- [x] src/AttendeeConfiguration.php
- [x] src/Attendee.php
- [x] src/SlotHandler

### CODE REVIEW

File: src/Attendee.php [hunk 1]
```diff
@@ -8,22 +8,39 @@
...
-
+    /**
+     * Google Service
+     * @var \Google_Service_Calendar
+     */
     private $googleService;
-
+    /**
+     * Handler for slot selection
+     * @var Appointment\SlotHandler
```
Comment (charsme) [inline src/Attendee.php]: namespace harusnya `\Appointment\SlotHandler` atau `SlotHandler` aja

File: src/Attendee.php [hunk 2]
```diff
@@ -8,22 +8,39 @@
...
+     */
     private $slotHandler;
-
+    /**
+     * Configuration
+     * including credential
+     * And application settings
+     * @var Appointment\AttendeeConfiguration
```
Comment (charsme) [inline src/Attendee.php]: sama kayak sebelumnya

**Comment by charsme** [comment]
Sama sekalian ada issues ini ken: https://scrutinizer-ci.com/g/KLYgarage/appointment/inspections/37d4406a-784e-4676-93df-d8174221922c/issues/files/src/EventConfiguration.php?status=new&orderField=path&order=asc&honorSelectedPaths=0

**Comment by kennykarnama** [comment]
sudah diperbaiki pak. bisa dicek @charsme

**Pull request merged**